doc/terminänderungen.md

```diff
@@ -0,0 +1,14 @@
+# Terminänderungen
+## Terminfindung
+- Abstimmung in einem Tool, welches zu diversen Terminen die Meinung der Leute abfragt.
+  - Abstimmzeit eine Woche  
+- Wenn kein eindeutig besserer Termin erkannt werden kann, sollte eine weitere Wahlabstimmung im Form einer Stichwahl durchgeführt werden.
+  - Abstimmzeit mindestens einen Tag  
+
+## Bekanntgabe
+Es muss an diveren Stellen der Termin angepasst werden:
+- Website
+- Kalender
+- Chatbeschreibung und als Nachricht im Chat
+- Veröffentlichung per Mail
+- Veröffentlichung per Twitter
```

Comment:
```suggestion
```

Weil nicht wirklich da relevant IMHO

Reply:
Wieso meinst du? Ja ich war mir bei dem Punkt auch nicht sicher, aber ich dachte mir gerade in der Pandemie is das vllt ganz sinnvoll.

Reply:
Aber auf Twitter? Finde ich ist nicht das richtige Medium dafür. Telegram, Mail… weiß nicht, Twitter finde ich komisch.